snmp/datadog_checks/snmp/data/default_profiles/barracuda-cloudgen.yaml

```diff
@@ -141,13 +141,14 @@ metrics:
           OID: 1.3.6.1.4.1.10704.1.7.1.2
           name: bgpNeighborState
         tag: bgp_neighbor_state
-        mapping:
-          1: idle
-          2: connect
-          3: active
-          4: opensent
-          5: openconfirm
-          6: established
+        # mapping:
+        #   1: idle
+        #   2: connect
+        #   3: active
+        #   4: opensent
+        #   5: openconfirm
+        #   6: established
+        #  mappings not found in MIB
```

Comment:
Huh, weird. The MIB has 
```
    bgpNeighborState OBJECT-TYPE
        -- SYNTAX NeighborState
        SYNTAX Integer32
```
so for some reason they *had* it as a proper neighborstate and then changed it to a plain integer at some point. These are pretty standard states, though - I'd be a little surprised if the device doesn't follow these states.

Reply:
I didn't find a reference to where these values are coming from

Reply:
The best I can find is https://campus.barracuda.com/product/cloudgenfirewall/doc/79463283/phion-mib-field-descriptions/#heading-7 , which lists the possible values as "Established, ..." so I'm pretty sure they intended for them to be the standard BGP states. These states are defined in appendix 1 of [RFC 1163](https://datatracker.ietf.org/doc/html/rfc1163)